Common/Gfx/Shape.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright (C) 2016 Rainmeter Project Developers
+ *
+ * This Source Code Form is subject to the terms of the GNU General Public
+ * License; either version 2 of the License, or (at your option) any later
+ * version. If a copy of the GPL was not distributed with this file, You can
+ * obtain one at <https://www.gnu.org/licenses/gpl-2.0.html>. */
+
+#ifndef RM_GFX_SHAPE_H_
+#define RM_GFX_SHAPE_H_
+
+#include <gdiplus.h>
```

Comment:
This include is unnecessary, but might want to include `<d2d1_1.h>` and `<wrl/client.h>` for correctness.